bahmni-emr-api/src/test/resources/TestingApplicationContext.xml

```diff
@@ -18,6 +18,11 @@
         <property name="mappingJarLocations">
             <ref bean="mappingJarResources"/>
         </property>
+        <property name="packagesToScan">
+            <list>
+                <value>org.openmrs</value>
```

Comment:
this is a very wide component scan? can't we narrow down?

Reply:
@angshu openmrs has suggested this fix to run IT's, where all the beans would be instantiated from openmrs context.